### ISSUE TITLE
Typos in Manual (on Inserttag page)

### ISSUE THREAD

**Issue by madmaharaja** [contao/docs#980]
I am not sure -- maybe I haven't understood it correctly, but it seems to me that there are some mistakes on the page regarding Inserttags as pointed out in the following screenshot.
![grafik](https://user-images.githubusercontent.com/3133710/151952145-ca287fd4-f48d-47e9-af4b-c0495dc28d8f.png)

- I assume that the first if**n**lng-Tag (Your name) is supposed to contain the language ISO code for "en" and not "de", right?
- In the second example for if**n**lng -- shouldn't it be "if**n**lng" instead of "iflng" (missing "n")?

See:
https://github.com/contao/docs/edit/main/docs/manual/article-management/insert-tags.de.md
https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/

**Comment by fritzmg** [comment]
No, that's deliberate. It's akin to:

```php
if ('de' !== $language) {
    echo 'Your Name';
} else {
    echo 'Ihr Name';
}
```

Personally I would write it the other way around, but it does not matter.

**Comment by madmaharaja** [comment]
Oh, ok, thank you.

- So in plain English "iflng::de" means: If page language **is** German, then display the following text.
- And "ifnlng::de" means: If page language is **not** German, then display the following text -- right?

Maybe it's just me, but in this case I don't find the description in the manual very easy to understand. I had to read it multiple times until I actually realised the difference in the two explanations (that in the first one/iflng it says "**nicht**"). 
Could one maybe highlight the "nicht" in the first explanation and, for the sake of readability, in the second explanation move the text in the brackets to the end of the sentence? -->
"Dieses Tag wird komplett entfernt, wenn die Sprache der Seite mit der Tag-Sprache übereinstimmt (siehe auch Tipp bei {{iflng}} für weitere Optionen)."
